### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by toppi420**
Update software android 11

- [ ] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by toppi420** [comment]
Android11

**Comment by Ba345** [comment]
Mình chào tất cả mình đến từ Việt Nam

**Comment by Ba345** [comment]
Phone 840948875417

**Comment by mutza974** [comment]
> Mình chào tất cả mình đến từ Việt Nam

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+---samsung android 11
 layout: index
 lang: en
----
+docs/content/github/writing-on-github/working-with-advanced-formatting.md.
```
Comment (SMAKSS) [inline index.html:4]: I'm not sure, what are you exactly trying to do, but this kind of change is totally unnecessary here.

**Review by mutza974 — APPROVED** [review]
@mutza974

File: index.html (hunk 1 above)
Comment (mutza974) [reply index.html:4]: @mutza974

**Review by mutza974 — APPROVED** [review]
@mutza974

File: index.html (hunk 1 above)
Comment (binhle113) [reply index.html:4]: +4

**Pull request closed without merging**